src/auro-menu.js

```diff
@@ -242,6 +242,31 @@ class AuroMenu extends LitElement {
     });
   }
 
+  /**
+   * Used to make a menuoption selection directly using a defined `value`.
```

Comment:
Definitions of functions shouldn't be in the past tense. 

```suggestion
   * Method to apply `selected` attribute to `menuoption` via `value`.
```

Reply:
I am seeing this as resolved, but not updated? Regardless of the final definition, the current definition should be re-worded. The use of a past tense makes this a confusing sentence.